test/api/buffer/receivers/tasks/001-get-ok/expected-response.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "abcdef",
+  "receiverId": "my-receiver",
+  "url": "https://buffer.keboola.com/v1/receivers/my-receiver/tasks/receiver.create/",
+  "type": "receiver.create",
+  "createdAt": "2023-01-01T08:02:06.000Z",
+  "finishedAt": "2023-01-01T09:02:06.000Z",
+  "duration": 3600,
+  "isFinished": true,
+  "result": "receiver created"
+}
```

Comment:
@michaljurecko Dává to takhle nějak smysl? Pro nejsnazší parsování key z payloadu bych dal URL `/v1/receivers/my-receiver/tasks/receiver.create/2023-01-01T08:02:06.000Z/abcdef`. Asi by bylo pěknější udělat z task key hash, ale to by se musel ukládat do etcd, resp. by se nejspíš musel dát rovnou do key etcd záznamu. Co myslíš?